src/main/java/com/codeborne/selenide/Conditional.java

```diff
@@ -6,6 +6,6 @@
 
   void should(Predicate<T> predicate, String message);
 
-  void should(Predicate<T> predicate, String message, int timeout);
+  void should(Predicate<T> predicate, String message, long timeout);
```

Comment:
I don't think that `int`/`long` meet selenide api.
For example methods with any sort of wait such `shouldBe` or `shouldBe` using `Duration` type of `timeout`.
Also we already deprecate methods `waitUntil` or `waitWhile` with `long` type `timeout` parameter.
Is is possible to use `Duration` instead of `long` in awaility library methods